<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIKanSoloCreateNewTask</class>
 <widget class="QDialog" name="UIKanSoloCreateNewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Task</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QTableWidget
{
	border:none;
	background:transparent;
}
QTableView::item
{
  border: 0px;
  padding: 8px;
}
QTableWidget::item{ background-color: transparent; }
QTableWidget::item:selected{ background-color: #aaaaff; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="m_txtTitle">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter Task Title...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Task Scores:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="m_tblScores">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="1" column="1">
        <widget class="QTextEdit" name="m_txtTask">
         <property name="placeholderText">
          <string>Enter Task Description...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="m_grpPriority">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Task Priority:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QTableWidget" name="m_tblPriority">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>48</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::NoPen</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resx/resources.qrc">
         <normaloff>:/imgDelete.png</normaloff>:/imgDelete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resx/resources.qrc">
         <normaloff>:/imgOk.png</normaloff>:/imgOk.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_txtTitle</tabstop>
  <tabstop>m_txtTask</tabstop>
  <tabstop>m_btnCancel</tabstop>
  <tabstop>m_btnSave</tabstop>
 </tabstops>
 <resources>
  <include location="../resx/resources.qrc"/>
 </resources>
 <connections/>
</ui>
